Command/GenerateAdminCommand.php

```diff
@@ -237,6 +250,42 @@ protected function interact(InputInterface $input, OutputInterface $output)
         $input->setOption('bundle', $bundleName);
     }
 
+    /**
+     * Returns the absolute path to the services config-dir
+     *
+     * @param string $bundleName
+     *
+     * @return string
+     */
+    private function determineConfigDir($bundleName)
+    {
+        $applicationConfigDir = $this->getKernel()->getRootDir() . '/config';
```

Comment:
this is incorrect. Use `$applicationConfigDir = $this->getKernel()->getRootDir() . '/../config';`